src/lib/sectionList/listViews/sectionListViewsConfig.ts

```diff
@@ -126,6 +126,19 @@ export const modelListViewsConfig = {
             default: ['dataDimensionType', 'categoryCombo'],
         },
     },
+    categoryOptionGroupSet: {
+        columns: {
+            default: [
+                'name',
+                'dataDimensionType',
+                DESCRIPTORS.publicAccess,
+                'lastUpdated',
+            ],
+        },
+        filters: {
+            default: ['dataDimensionType', 'categoryCombo'],
```

Comment:
`categoryCombo` is not a reference in `categoryOptionGroupSet`. See [schema](https://debug.dhis2.org/dev/api/schemas/categoryOptionGroupSet)
```suggestion
            default: ['dataDimensionType'],
```

We could add `categoryOptionGroup` filter - however that would need to add a new handler for that filter. 
You can take a look at this commit: https://github.com/dhis2/maintenance-app-beta/commit/a5c7ac5613fa87afb4100aff26b0901c4262509c , if you want to introduce that filter now - but not crucial at this time.